src/main/java/com/kosherjava/zmanim/hebrewcalendar/JewishCalendar.java

```diff
@@ -240,6 +240,47 @@ public boolean isBirkasHachamah() {
 		return false;
 	}
 
+	/**
+	 * Tekufas Shmuel: a solar year is 365.25 days.
+	 * notation: days,hours,chalakim
+	 * molad BaHaRad was 2D,5H,204C
+	 * or 5H,204C from the start of rosh hashana year 1
+	 * molad nissan add 177D,4H,438C (6 * 29D,12H,793C)
+	 * or 177D,9H,642C after rosh hashana year 1
+	 * tekufas nissan was 7D,9H,642C before molad nissan ~rambam.
+	 * or 170D,0H,0C after rosh hashana year 1
+	 * tekufas tishrei was 182D,3H (365.25 / 2) before tekufas nissan
+	 * or 12D,15H before Rosh Hashana year 1
+	 * outside of EY we say תל ומטר in ברכת השנים from 60 days after tekufas tishrei.
```

Comment:
@plonibarploni Thanks. Please chnage the Hebrew chars to Birkas Hashanim and Tal Umatar. The charset for the files are not gauranteed to be UTF-8 everywhere.

Reply:
Fixed